security/libu2f-server/files/patch-core-macro-fix.diff

```diff
@@ -0,0 +1,22 @@
+--- u2f-server/core.c.orig	2017-12-18 03:52:12.000000000 -0800
```

Comment:
Usually we would add references at the beginning of the patch file, or use `git format-patch` format for proper attribution.

Reply:
Would it suffice to provide attribution as follows?
```
# Patch taken from https://github.com/Yubico/libu2f-server/commit/f7c4983
--- u2f-host/u2fmisc.c.orig	2019-02-22 03:31:45.000000000 -0800
+++ u2f-host/u2fmisc.c	2021-08-12 23:06:00.000000000 -0700
...
```

Reply:
```
https://github.com/Yubico/libu2f-server/commit/f7c4983b31909299c47bf9b2627c84b6bfe225de
--- u2f-host/u2fmisc.c.orig	2019-02-22 03:31:45.000000000 -0800
+++ u2f-host/u2fmisc.c	2021-08-12 23:06:00.000000000 -0700
...
```

would do. You don't need the `#` because `patch` will ignore any unrecognized headers.

Reply:
Thank you for the advice. I will also do this in the patch of the libu2f-host PR.